cross/android/arm64/tryrun.cmake

```diff
@@ -0,0 +1,128 @@
+# Need to verify this
```

Comment:
I don't think tryrun is required for core-setup. You copied the values from coreclr which are not needed here.